libraries/DataFlash/LogStructure.h

```diff
@@ -84,6 +84,7 @@ const struct UnitStructure log_Units[] = {
     { 'G', "Gauss" },         // Gauss is not an SI unit, but 1 tesla = 10000 gauss so a simple replacement is not possible here
     { 'h', "degheading" },    // 0.? to 359.?
     { 'i', "A.s" },           // Ampere second
+    { 'J', "W.s" },           // Joule (Watt second)
```

Comment:
Why is this not Wh? That seems like the most common unit for considering power consumption?

Reply:
Unit is "Watt seconds"  to keep consistent with Ampere seconds for capacity. 

 "J" is only used as MultiplierStructure.ID, which is, by pure coincidence, similar to Joule, which is, by another coincidence, one Watt times one second ;)

Reply:
Sorry, I'm not quite following yet! I don't even see the Ampere second being used anywhere? CurTot is currently not tagged with a unit. If you think it's a better unit for log analysis that's fine, but given that we are just dumping a Wh number directly already as a float and it's a reasonable unit to accumulate energy consumption I'm curious why we would want to request GCS's scale it, when not scaling will leave us with a valid result, and graph correctly in those that don't scale.